code/game/objects/items/fishingrod.dm

```diff
@@ -1,7 +1,7 @@
 GLOBAL_LIST_INIT(fish_rates, list(
 	/obj/item/fishy/carp		=9,
 	/obj/item/fishy/salmon		=9,
-	/obj/item/fishy/eel			=2,
+	/obj/item/fishy/eel			=0,
```

Comment:
Brother, don't leave it at 0, just comment it out

Reply:
If you comment it out it’s more difficult to restore if needed. I don’t like deleting original code from tg or cit or wherever it’s from.

Reply:
> If you comment it out it’s more difficult to restore if needed. I don’t like deleting original code from tg or cit or wherever it’s from.

Comment it out or remove it, it won't be merged as is.

Reply:
> > If you comment it out it’s more difficult to restore if needed. I don’t like deleting original code from tg or cit or wherever it’s from.
> 
> Comment it out or remove it, it won't be merged as is.

OK done